src/tools/quemap/brush.c

```diff
@@ -43,7 +43,7 @@ static void BoundBrush(bsp_brush_t *brush) {
 /**
  * @brief
  */
-static void CreateBrushWindings(bsp_brush_t *brush) {
+static void CreateBrushWindings(qbsp_brush_t *brush) {
```

Comment:
I only say that because Quemap suffers from short, nondescript naming pretty much throughout ;D